pkg/tfbridge/diff.go

```diff
@@ -47,7 +47,9 @@ func containsComputedValues(v resource.PropertyValue) bool {
 	}
 }
 
-// makePropertyDiff checks the given property for diffs and records any that are found in diff.
+type propertyVisitor func(attributeKey, propertyPath string, value resource.PropertyValue) bool
```

Comment:
Could you comment what the semantics of the return value are of this function?

Reply:
Done. Thank you :)